### DISCUSSION TITLE
AI是否會提供錯誤資訊(像是現在的以巴戰爭)??

### DISCUSSION THREAD

**Discussion by Fion-Lin** [1101-GS4719-C/Python-Community-GS4719-C#1655] [General]
Recently, the conflict between Israeli and Palestine is breaking out. However, if you ask AI chatbots about the information of this conflict, it turns out that they mostly provide wrong information. The content in the below link compares multiple different online AI chatbots to see the result coming out from the computer itself. The reason why the information are mostly untrue is that most of the AI chatbot models haven't been updated yet, so this can release false information and somehow mislead users. BTW, I'm trying to learn about some historical events and relationships between these two countries to know more about the conflict from the third-person view.

Here's the link: 
https://www.businessinsider.com/ai-chatbots-israel-hamas-conflict-google-bard-microsoft-openai-chatgpt-2023-10

**Comment by v0ouo** [comment]
常常遇見AI會提供錯誤的訊息吧，我平常有時候遇到不會的東西去問CHATGPT，結果他常常連基本的東西都回答錯誤，有時候要自己去糾正它，它才會說我錯了，然後糾正自己剛剛的錯誤，所以AI提供的資訊還是要自己判斷。

**Comment by 1chooo** [comment]
不過現在 ChatGPT 的資料庫應該沒有更新到現在的以巴戰爭吧！加上他對某些關鍵詞會失靈，說不定戰爭也被他歸在比較殘忍、不道義的那部分，所以會失靈，但我比較覺得會錯的原因是他的資料根本沒有這部分，所以一切都是自己亂想的！

**Comment by Chengbosco** [comment]
gpt目前資料只到2021，所以很多東西都只能說不知道，不然就會胡謅一堆根本不相干的東西，像之前做國文作業要找一個作者，阿書籍名稱是{槍}，結果他跟我說什麼他是槍擊要犯:)，所以資料也要自己判斷，看一下是不是真的，也有讓gpt更新資料庫的方法(https://www.storm.mg/article/4783101)，但我沒用過。

**Comment by ATREE01** [comment]
語言模型機本事就是透過訓練之後讓他判斷下一個字要接甚麼會比較合理，所以如果像是遇到數學運算或是一些邏輯問題可能語句看起來很正常但是內容會很怪，尤其是資料庫沒有的東西就只是在玩文字接龍的感覺。
